config/routes.rb

```diff
@@ -41,6 +41,7 @@
         get "display_purchases", :as => "display_purchases"
         get "display_books", :as => "display_books"
         get "display_book", :as => "display_book"
+        post "/toggle_flag", :to =>"dashboard#toggle_flag"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.